debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/MongoDbReplicaSet.java

```diff
@@ -368,6 +370,11 @@ public List<String> getHostNames() {
                 .collect(Collectors.toList());
     }
 
+    public MongoDbReplicaSet withStartupTimeout(Duration startupTimeout) {
```

Comment:
The entire class is not designed as a fluent builder. I don't see a reason why a single message should behave differently. This should be in the builder. Kindly place it there and be careful about mixing various design patterns into single thing.

Thank you.

Reply:
@jcechace This just goes back to your own comment here:
https://github.com/debezium/debezium/blob/main/debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/testhelper/RestExtensionTestInfrastructure.java#L135-L140

The Oracle Connect REST extension PR will then use this method instead of the code linked, but I decided to add it to this bugfix PR so it can be used early before we finish that other PR. Maybe you want to log a Jira to change and refactor this part/class?

Reply:
@rk3rn3r  There is a structure to those classes. Put it into the builder, pass it from the builder to the RS class and then over to the underlying containers in a similar fashion. 

I don't see a need for Jira. The code was not refactored correctly and the PR was merged before I could review it. Please make it consistent with the rest of the class design. Thank you.

Reply:
@jcechace This is not how we work and I only applied a change that you suggested in the comment that I linked above, which was a good comment. The class is `Startable` and suggested to reflect a running container. Thus, a timeout config makes sense. Feel free to fix it that it fits your individual idea of that class or provide a Jira explaining what you want to achieve.

Reply:
https://issues.redhat.com/browse/DBZ-7054